suse/rdma.sriov-service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Initialize SRIOV for RDMA devices
+RefuseManualStop=true
+DefaultDependencies=false
+Conflicts=emergency.target emergency.service
+Before=network.target remote-fs-pre.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/lib/rdma-set-sriov-vf
+
+[Install]
+WantedBy=sysinit.target
```

Comment:
Rules triggered by udev should not be installed, otherwise they run unconditionally

Reply:
Noted. Will fix